assets/js/workflows/usb.js

```diff
@@ -246,11 +248,12 @@ class USBWorkflow extends Workflow {
         // At this point we should see if we should init the file client and check if have a saved dir handle
         this.initFileClient(new FileTransferClient(this.connectionStatus.bind(this), this._uid));
         const fileClient = this.fileHelper.getFileClient();
-        console.log("Got file client");
         const result = await fileClient.loadSavedDirHandle();
-        console.log(result);
         if (result) {
+            console.log("Successfully loaded directory:", fileClient.getWorkingDirectoryName());
             this._hostFolderChanged();
```

Comment:
If you're making _hostFolderChanged() asynchronous, please add an await before `this._hostFolderChanged();`

Reply:
Done.